packages/grafana-ui/src/themes/GlobalStyles/elements.ts

```diff
@@ -120,8 +120,17 @@ export function getElementStyles(theme: GrafanaTheme2) {
       color: ${theme.colors.text.primary};
       text-decoration: none;
 
-      &:focus {
+      &:focus,
+      &:focus-visible {
+        outline: 2px dotted transparent;
+        outline-offset: 2px;
+        box-shadow: 0 0 0 2px ${theme.colors.background.canvas}, 0 0 0px 4px ${theme.colors.primary.main};
+        transition: all 0.2s cubic-bezier(0.19, 1, 0.22, 1);
+      }
+
+      &:focus:not(:focus-visible) {
         outline: none;
+        box-shadow: none;
```

Comment:
What's the reasoning for this?

Reply:
I just added this from other focus styles. do we need to add styles for `focus:not(:focus-visible)` ?